go-apps/meep-app-enablement/server/app-support/app-support.go

```diff
@@ -144,6 +148,11 @@ func Init(globalMutex *sync.Mutex) (err error) {
 func reInit() {
 	//next available subsId will be overrriden if subscriptions already existed
 	nextSubscriptionIdAvailable = 1
+
+	baseKey := appEnablementBaseKey + ":apps:*:" + mappsupportKey
+	keyName := baseKey + ":subscriptions:" + "*"
```

Comment:
No need for "subscriptions" level in key. Simply use this key:
`keyName := appEnablementBaseKey + ":apps:*:" + subscriptionsKey + "*"`

Reply:
used on one line